RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -210,7 +55,7 @@ aggregate autoMaximize(string req, int maxPrice, int priceLevel, boolean simulat
 	return maxrecord;
 }
 
-void debugMaximize(string req, int meat)
+void debugMaximize(string req, int meat)	//This function will be removed.
```

Comment:
this comment already existed in the public prototype section. I just moved it here